contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -316,10 +321,48 @@ else if (build && push && dataDirs.size() != 1) {
         }
         if ((!build && push) || (build && !push)) {
             log.warn("DEPRECATED : Creating one build job and separate push job is a deprecated strategy. Instead create"
-                    + "just one job with both build and push set as true and pass a list of cluster urls.");
+                     + "just one job with both build and push set as true and pass a list of cluster urls.");
         }
     }
 
+    private String getMatchingServerSupportedCompressionCodec(int nodeId) {
+        AdminClient adminclient = new AdminClient(clusterUrl.get(0),
```

Comment:
We tend to keep the clusters in symmetry. So dint bother about checking all clusters. Anyway I will ask the SREs on what they think.